pymc3/math.py

```diff
@@ -1,15 +1,13 @@
 from __future__ import division
-from theano.tensor import constant, flatten, zeros_like, ones_like, stack, concatenate, sum, prod, lt, gt, le, ge, eq, \
-    neq, switch, clip, where, and_, or_, abs_
-from theano.tensor import exp, log, cos, sin, tan, cosh, sinh, \
-    tanh, sqr, sqrt, erf, erfinv, dot
-from theano.tensor import maximum, minimum, sgn, ceil, floor
-from theano.tensor.nlinalg import det, matrix_inverse, \
-    extract_diag, matrix_dot, trace
-from theano.tensor.nnet import sigmoid
+import sys
 import theano
 import theano.tensor as tt
-import sys
+from theano.tensor import constant, flatten, zeros_like, ones_like, stack, concatenate, sum, prod, \
```

Comment:
nit-picky: if you do `from theano.tensor import (...)` you don't need the escape characters.